.github/workflows/regenerate-snapshots.yml

```diff
@@ -0,0 +1,92 @@
+name: Regenerate Textual snapshots from a comment
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  regenerate-snapshots:
+    # Only run if comment is on a PR with the main repo, and if it contains the magic keywords
+    if: >
+      contains(github.event.comment.html_url, '/pull/') &&
+      contains(github.event.comment.body, '@nf-core-bot regenerate snapshots') &&
+      github.repository == 'nf-core/tools'
+    runs-on: ubuntu-latest
+    steps:
+      # Use the @nf-core-bot token to check out so we can push later
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
+        with:
+          token: ${{ secrets.nf_core_bot_auth_token }}
+
+      # indication that the command is running
+      - name: React on comment
+        uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043 # v4
+        with:
+          comment-id: ${{ github.event.comment.id }}
+          reactions: eyes
+
+      # Action runs on the issue comment, so we don't get the PR by default
+      # Use the gh cli to check out the PR
+      - name: Checkout Pull Request
+        run: gh pr checkout ${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.nf_core_bot_auth_token }}
+
+      # Install dependencies and run pytest
+      - name: Set up Python
+        uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
+        with:
+          python-version: "3.12"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip -r requirements-dev.txt
+          pip install -e .
+
+      - name: Run pytest to regenerate snapshots
+        id: pytest
+        run: |
+          python3 -m pytest tests/test_create_app.py --snapshot-update --color=yes --cov --durations=0
+        continue-on-error: true
+
+      # indication that the run has finished
+      - name: react if finished succesfully
+        if: steps.pytest.outcome == 'success'
```

Comment:
Don't remember now, does pytest report a success if only snapshots are updated or is also a failure because something changed?

Reply:
I think it's success, because it also runs the tests